firefox-ios/RustFxA/FirefoxAccountSignInViewController.swift

```diff
@@ -282,7 +282,7 @@ class FirefoxAccountSignInViewController: UIViewController, Themeable {
                     // password
                     scopes: [OAuthScope.profile, OAuthScope.oldSync, OAuthScope.session]
                 ) { [weak self] result in
-                    guard let self = self else { return }
+                    guard self != nil else { return }
```

Comment:
so I can better understand, why do we need to weakify self here?

Reply:
Not sure. I just fixed the warning, honestly, because we were getting close to the threshold.